projects/objects/balls/protos/FifaSoccerBall.proto

```diff
@@ -0,0 +1,96 @@
+#VRML_SIM R2021a utf8
+# license: Apache License 2.0
+# license url: http://www.apache.org/licenses/LICENSE-2.0
+# A black and white FIFA soccer ball with customizable size.
+# This object is physics-enabled so it will roll or bounce when hit.
+# tags: static
+
+PROTO FifaSoccerBall [
+  field SFVec3f    translation 0 0 0
+  field SFRotation rotation    0 1 0 0
+  field SFString   name        "fifa soccer ball"
+  field SFInt32    size        5 # Official FIFA ball sizes ranging from 1 to 5.
+]
+{
+  %{
+    local size = fields.size.value
+    if size < 1 then
+      size = 1
+      io.stderr:write("'size' must be strictly positive\n")
+    elseif size > 5 then
+      size = 5
+      io.stderr:write("'size' must be less than or equal to 5\n")
+    end
```

Comment:
That looks like a perfect use case for field value restriction instead of custom check 😉 
https://cyberbotics.com/doc/reference/proto-definition#field-value-restriction

Note also (https://cyberbotics.com/doc/reference/proto-design-guidelines#field-comments):

> In general, field comments should use a conjugated verb (in most cases "Defines") and end with period (".").

Reply:
Thanks! I will fix this.